backend/de.metas.cucumber/src/test/java/de/metas/cucumber/stepdefs/issue/AD_Issue_StepDef.java

```diff
@@ -0,0 +1,126 @@
+/*
+ * #%L
+ * de.metas.cucumber
+ * %%
+ * Copyright (C) 2021 metas GmbH
+ * %%
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as
+ * published by the Free Software Foundation, either version 2 of the
+ * License, or (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public
+ * License along with this program. If not, see
+ * <http://www.gnu.org/licenses/gpl-2.0.html>.
+ * #L%
+ */
+
+package de.metas.cucumber.stepdefs.issue;
+
+import com.fasterxml.jackson.databind.ObjectMapper;
+import de.metas.common.rest_api.JsonError;
+import de.metas.common.rest_api.JsonErrorItem;
+import de.metas.common.rest_api.issue.JsonCreateIssueResponse;
+import de.metas.cucumber.stepdefs.APIResponse;
+import de.metas.cucumber.stepdefs.RESTUtil;
+import de.metas.error.AdIssueId;
+import de.metas.organization.IOrgDAO;
+import de.metas.organization.OrgId;
+import de.metas.process.PInstanceId;
+import de.metas.util.Services;
+import io.cucumber.java.en.Given;
+import io.cucumber.java.en.Then;
+import io.cucumber.java.en.When;
+import org.adempiere.ad.dao.IQueryBL;
+import org.adempiere.model.InterfaceWrapperHelper;
+import org.compiere.model.I_AD_Issue;
+import org.compiere.model.I_AD_Org;
+import org.compiere.model.I_AD_PInstance;
+
+import java.io.IOException;
+import java.util.List;
+
+import static org.assertj.core.api.Assertions.*;
+
+public class AD_Issue_StepDef
+{
+	private PInstanceId pInstanceId;
+	private String userAuthToken;
+	private APIResponse apiResponse;
+	private String apiRequest;
+
+	private final IQueryBL queryBL = Services.get(IQueryBL.class);
+	private final IOrgDAO orgDAO = Services.get(IOrgDAO.class);
+
+	public static final String endpointPath = "api/process/$pInstanceId/externalstatus/error";
+
+	@Given("the existing user has a valid API token")
+	public void the_existing_user_has_the_authtoken() throws IOException
+	{
+		userAuthToken = RESTUtil.getAuthToken("metasfresh", "WebUI");
+	}
+
+	@When("the metasfresh REST-API endpoint for issue creation receives a request with the payload")
+	public void metasfresh_rest_api_endpoint_api_external_ref_receives_get_request_with_the_payload(
+			final String payload) throws IOException
+	{
+		apiRequest = payload;
+		apiResponse = RESTUtil.performHTTPRequest(endpointPath.replace("$pInstanceId", String.valueOf(pInstanceId.getRepoId())),
```

Comment:
In the next(!) issue, please change this to use the `REST_API_StepDef` for doing the rest-call.
We can inject APIResponse in both REST_API_StepDef instance and `AD_Issue_StepDef`  to change the API's responce (see the other step_defs on to inject a shared-state-instance).